src/config/settings/base.py

```diff
@@ -82,6 +83,24 @@
     "allauth",
     "allauth.account",
     "allauth.socialaccount",
+    # https://django-machina.readthedocs.io/en/latest/getting_started.html#django-settings
+    # Django-machina dependencies:
+    "mptt",
+    "haystack",
+    "widget_tweaks",
+    # https://django-machina.readthedocs.io/en/latest/getting_started.html#django-settings
+    # Django-machina apps:
```

Comment:
This is very minor, could you swap the order of the title and link?

ie:

```
# Django-machina apps:
# https://django-machina.readthedocs.io/en/latest/getting_started.html#django-settings
```

This is because titling comments like this conceptually signal that everything below it is related to this title. 

By the link appearing above the title feels like the links are relevant for the code *above* not below the title.

Reply:
Done.